ui/src/components/helpers/userInformationHelpers.js

```diff
@@ -73,7 +74,7 @@ const userDataCleanup = (data) => {
 };
 
 const getUserDataRequest = async (uid) => {
-  return await axios.post("/api/users/get", {
+  return await axios.get("/api/users/get", {
     userId: uid,
```

Comment:
this will need to be
`return await axios.get("/api/users/get", {
    params: { userId: uid }})`